docs/quick-lint-js.schema.config

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
Nit: According to the `.editorconfig` file, `.json` files should be 2-space indented. Please reindent this file.